FrEIA/modules/splines/rational_quadratic.py

```diff
@@ -44,6 +45,65 @@ def _spline2(self, x: torch.Tensor, parameters: Dict[str, torch.Tensor], rev: bo
         return rational_quadratic_spline(x, left, right, bottom, top, deltas_left, deltas_right, rev=rev)
 
 
+class RationalQuadraticSpline_1D(BinnedSpline):
```

Comment:
I think the name might be misleading… 1D suggests that one only can have 1D transformations, but this is not the case

Reply:
Renamed to ElementwiseRationalQuadraticSpline

Reply:
Looks good to me!